portfolio/src/main/webapp/style.css

```diff
@@ -4,6 +4,45 @@
   width: 650px;
 }
 
+intro-title {
+    font-family: cursive;
+}
+
+greeting-button {
+  background-color: rgb(151, 91, 201);
+  border: none;
+  color: white;
+  padding: 15px ;
+  text-align: center;
+  font-size: 20px;
+  margin: 4px 2px;
+}
+
+cursive {
+  font-family: cursive;
+}
+
 #greeting-container {
-  margin-top: 20px;
+  margin-top: 30px;
+  font-size: 25px;
+}
+
+div.gallery {
```

Comment:
Nit: you can probably just use `.gallery` without the `div`. I'd only add the `div` if you had a reason to need that much specificity.